### PR TITLE
Remove X from triangular params

### PR DESCRIPTION
**Pull request by dafeda**
## Pre review checklist

- [ ] Read through the code changes carefully after finishing work
- [ ] Make sure tests pass locally (after every commit!)
- [ ] Prepare changes in small commits for more convenient review (optional)
- [ ] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Updated documentation
- [ ] Ensured that unit tests are added for all new behavior (See 
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [ ] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: docs/reference/configuration/data_types.rst [hunk 1]
```diff
@@ -226,9 +226,9 @@ TRIANGULAR
 
 ::
 
-    TRIANGULAR XMIN XMODE XMAX
```
Comment (oyvindeide) [inline docs/reference/configuration/data_types.rst:229]: Also needs some updates here:
https://github.com/equinor/ert/blob/695cd506c1feb133eae09e28f7a5efc0c0b34be9/docs/reference/configuration/data_types.rst?plain=1#L365-L395

**Review by oyvindeide — APPROVED** [review]
LGTM, just a small comment about some docs update.

File: src/ert/config/gen_kw_config.py [hunk 2]
```diff
@@ -342,7 +342,7 @@ def _parse_transfer_function(param_string: str) -> TransferFunction:
             "NORMAL": ["MEAN", "STD"],
             "LOGNORMAL": ["MEAN", "STD"],
             "TRUNCATED_NORMAL": ["MEAN", "STD", "MIN", "MAX"],
-            "TRIANGULAR": ["XMIN", "XMODE", "XMAX"],
```
Comment (kvashchuka) [inline src/ert/config/gen_kw_config.py:345]: in this file there is a function called `trans_triangular`, which still uses variables `_xmin, _xmax, _xmode`. Should we rename those too for the naming consistency?

Comment (dafeda) [reply src/ert/config/gen_kw_config.py:345]: Sure 👍

**Pull request merged**